proposals/lip-0063.md

```diff
@@ -218,50 +218,72 @@ ffffff
    <td>The address used to store all tokens of legacy accounts.
    </td>
   </tr>
+  <tr>
+   <td><code>Q96_ZERO</code>
+   </td>
+   <td>bytes
+   </td>
+   <td><code>''</code>
+   </td>
+   <td>The empty byte string, which represents the zero in Q96 representation.
+   </td>
+  </tr>
```

Comment:
In the table above, the constant `LOCAL_ID_LSK` can be removed as it does not seemed to be used in this LIP any more.

Reply:
Further note that `TOKEN_ID_LSK` is different for the Betanet, Testnet and Mainnet mainchain.

Reply:
I removed the two constants. 

Regarding `TOKEN_ID_LSK`: I would leave this point open until https://github.com/LiskHQ/discussions/discussions/63 is solved. As this issue is spread over many LIPs, I would also prefer to fix it in a separate PR in which we fix all LIPs at the same time.

Reply:
Okay, sounds good.